.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           - {VERSION: "3.9", TOXENV: "pep8,packaging", COVERAGE: "false"}
           - {VERSION: "pypy2", TOXENV: "pypy-nocoverage", COVERAGE: "false"}
           - {VERSION: "pypy3", TOXENV: "pypy3-nocoverage", COVERAGE: "false"}
+          - {VERSION: "3.9", TOXENV: "docs", COVERAGE: "false"}
```

Comment:
Can we just run this in the ppe8/packaging build?

Reply:
We could. Do we want to have one job overloaded with three tasks? Is there a disadvantage to running docs separately?

Reply:
fewer builds, since there's a small overhead per build, and all 3 of these tox jobs are fast, make sense to bundle them